rotkehlchen/tests/api/test_snapshots.py

```diff
@@ -313,66 +315,66 @@ def assert_csv_export_response(response, csv_dir, main_currency: Asset, is_downl
 
 
 def test_export_snapshot(rotkehlchen_api_server, tmpdir_factory):
-    conn = rotkehlchen_api_server.rest_api.rotkehlchen.data.db.conn
-    ts = Timestamp(ts_now())
+    dbhandler = rotkehlchen_api_server.rest_api.rotkehlchen.data.db
+    ts = ts_now()
     csv_dir = str(tmpdir_factory.mktemp('test_csv_dir'))
     csv_dir2 = str(tmpdir_factory.mktemp('test_csv_dir2'))
-    _populate_db_with_balances(conn, ts)
-    _populate_db_with_location_data(conn, ts)
+    _populate_db_with_balances(dbhandler, ts)
+    _populate_db_with_location_data(dbhandler, ts)
 
     rotkehlchen_api_server.rest_api.rotkehlchen.data.db.set_settings(ModifiableDBSettings(main_currency=A_EUR))  # noqa: E501
-    response = requests.post(
+    response = requests.get(
         api_url_for(
             rotkehlchen_api_server,
-            'dbsnapshotexportingresource',
+            'per_timestamp_db_snapshots_resource',
+            timestamp=ts,
+            path=csv_dir,
+            action='export',
         ),
-        json={
-            'timestamp': ts,
-            'path': csv_dir,
-        },
     )
     assert_csv_export_response(response, csv_dir, main_currency=A_EUR, is_download=False)
 
     rotkehlchen_api_server.rest_api.rotkehlchen.data.db.set_settings(ModifiableDBSettings(main_currency=A_ETH))  # noqa: E501
-    response = requests.post(
+    response = requests.get(
         api_url_for(
             rotkehlchen_api_server,
-            'dbsnapshotexportingresource',
+            'per_timestamp_db_snapshots_resource',
+            timestamp=ts,
+            path=csv_dir2,
+            action='export',
         ),
-        json={
-            'timestamp': ts,
-            'path': csv_dir2,
-        },
     )
     assert_csv_export_response(response, csv_dir2, main_currency=A_ETH, is_download=False)
 
     rotkehlchen_api_server.rest_api.rotkehlchen.data.db.set_settings(ModifiableDBSettings(main_currency=A_USD))  # noqa: E501
-    response = requests.post(
+    response = requests.get(
         api_url_for(
             rotkehlchen_api_server,
-            'dbsnapshotexportingresource',
+            'per_timestamp_db_snapshots_resource',
+            timestamp=ts,
+            action='export',
         ),
-        json={
-            'timestamp': ts,
-            'path': csv_dir2,
-        },
     )
-    assert_csv_export_response(response, csv_dir2, main_currency=A_USD, is_download=False)
+    assert_error_response(
+        response,
+        contained_in_msg='A path has to be provided when action is export',
+    )
 
 
 def test_download_snapshot(rotkehlchen_api_server):
-    conn = rotkehlchen_api_server.rest_api.rotkehlchen.data.db.conn
-    ts = Timestamp(ts_now())
-    _populate_db_with_balances(conn, ts)
-    _populate_db_with_location_data(conn, ts)
+    dbhandler = rotkehlchen_api_server.rest_api.rotkehlchen.data.db
```

Comment:
I saw these also while rebasing and it looked a bit odd - naming wise. Can you just name it `db`? I know we don't like short names but in this case it's better. It was the only place in the entire rebase I had to do `dbhandler.user_write()` instead of `db.user_write()`

Change for the entire file.